Task-2/task-2.js

```diff
@@ -0,0 +1,52 @@
+const fs = require("fs");
+const { Transform } = require("stream");
+
+const sourcePath = "source.json";
+const destinationPath = "destination.json";
+
+const transformStream = new Transform({
+  transform(chunk, encoding, callback) {

```

Comment:
Here you can use `objectMode: true` in order to work with other js types such as object. Please read about it and modify code.